src/components/Toggle/Toggle.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../styles/states";
+@import "../../styles/global-css-settings";
+
+/* in the new toggle we drew. */
```

Comment:
automation removed part of the comment:
// Since it is not possible to change the design of the checkbox according to the storybook toggle requirements using css,
  // we hide the checkbox and draw a new one instead.
  // In order to allow accessibility, all operations will be performed on the hidden checkbox and be reflected
  // in the new toggle we drew.

Reply:
Fixed